lib/core/core.js

```diff
@@ -78,8 +89,22 @@ axe._thisWillBeDeletedDoNotUse.public = {
 axe._thisWillBeDeletedDoNotUse.utils =
   axe._thisWillBeDeletedDoNotUse.utils || {};
 axe._thisWillBeDeletedDoNotUse.utils.cacheNodeSelectors = cacheNodeSelectors;
-axe._thisWillBeDeletedDoNotUse.utils.getNodesMatchingExpression = getNodesMatchingExpression;
+axe._thisWillBeDeletedDoNotUse.utils.getNodesMatchingExpression =
+  getNodesMatchingExpression;
 axe._thisWillBeDeletedDoNotUse.utils.convertSelector = convertSelector;
+axe._thisWillBeDeletedDoNotUse.commons =
+  axe._thisWillBeDeletedDoNotUse.commons || {};
+axe._thisWillBeDeletedDoNotUse.commons.dom =
+  axe._thisWillBeDeletedDoNotUse.commons.dom || {};
+axe._thisWillBeDeletedDoNotUse.commons.dom.nativelyHidden = nativelyHidden;
+axe._thisWillBeDeletedDoNotUse.commons.dom.displayHidden = displayHidden;
+axe._thisWillBeDeletedDoNotUse.commons.dom.visibilityHidden = visibilityHidden;
+axe._thisWillBeDeletedDoNotUse.commons.dom.ariaHidden = ariaHidden;
+axe._thisWillBeDeletedDoNotUse.commons.dom.opacityHidden = opacityHidden;
+axe._thisWillBeDeletedDoNotUse.commons.dom.scrollHidden = scrollHidden;
+axe._thisWillBeDeletedDoNotUse.commons.dom.overflowHidden = overflowHidden;
+axe._thisWillBeDeletedDoNotUse.commons.dom.clipHidden = clipHidden;
+axe._thisWillBeDeletedDoNotUse.commons.dom.areaHidden = areaHidden;
```

Comment:
This is getting messy. Lets pull this into a separate file.

Reply:
I will add this to the tech debt ticket https://github.com/dequelabs/axe-core/issues/3655